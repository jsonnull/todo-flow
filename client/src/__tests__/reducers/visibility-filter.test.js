// @flow

import * as actions from '../../actions';
import visibilityFilter from '../../reducers/visibility-filter';

describe('visibilityFilter reducer', () => {
  it('should handle SET_VISIBILITY_FILTER', () => {
    const initialState = 'SHOW_ALL';

    expect(
      visibilityFilter(initialState, actions.setVisibilityFilter('SHOW_ACTIVE'))
    ).toMatchSnapshot();
  });
});
