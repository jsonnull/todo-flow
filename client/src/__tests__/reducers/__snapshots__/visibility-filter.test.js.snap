exports[`visibilityFilter reducer should handle SET_VISIBILITY_FILTER 1`] = `"SHOW_ACTIVE"`;
